zubhub_frontend/zubhub/src/views/activity_details/ActivityDetailsV2.jsx

```diff
@@ -139,8 +142,14 @@ export default function ActivityDetailsV2(props) {
         </div>
 
         <div style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between', marginTop: 32, gap: 10 }}>
-          <CustomButton primaryButtonOutlinedStyle style={{ borderRadius: 4 }}>
-            Create this Project
+          <CustomButton
+              primaryButtonOutlinedStyle
+              style={{ borderRadius: 4 }}
+              onClick={() => {
```

Comment:
`onClick={handleCreateProject}` should also be enough here since we are passing any arguments